test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -44,7 +44,7 @@
     torch.nn.GELU(approximate="tanh"): 10,
     torch.nn.ReLU6(): 3,
     torch.nn.SiLU(): 3,
-    torch.nn.Hardsigmoid(): 5,
+    # torch.nn.Hardsigmoid(): 5,
```

Comment:
why comment it out?

Reply:
Hardsigmoid will be decomposed into add, clamp_min, clamp_max, div. With this pr, scalar add will be folded with conv/linear, which will change the behavior of the following fusion. I haven't figured out how to fix this issue yet, do you have any ideas?

Reply:
I currently only support the float type scalar case to avoid this problem.